src/js/components/Settings/VoterEmailAddressEntry.jsx

```diff
@@ -498,7 +516,11 @@ class VoterEmailAddressEntry extends Component {
                   color="primary"
                   disabled={disableEmailVerificationButton || signInCodeEmailSentAndWaitingForResponse}
                   id="voterEmailAddressEntrySendCode"
-                  onClick={this.sendSignInCodeEmail}
+                  onClick={(e) => {
+                    const buttonId = e.target.id;
+                    this.sendSignInCodeEmail();
+                    this.pushDataLayer(buttonId, 'sendVerification');
```

Comment:
In this case I added 'sendVerification' to the Properties spreadsheet: https://docs.google.com/spreadsheets/d/1KO4_AOmddKvIpA79XrZdaVBdRkcP1HitqrvQkCo2A34/edit?pli=1&gid=1828322090#gid=1828322090